### PR TITLE
Fixing Attendance issue and Style Changes

### LINKED ISSUE #177: username and profile picture are broken on meeting attendance page
screenshot from 2018-04-10 20-55-06

### LINKED ISSUE #178: Dropdown Arrow does not Expand Information Box in Firefox
When I click the dropdown arrow on any page in Firefox, it does not expand the information box contained.

### PR DESCRIPTION
**Pull request by devinmatte**
Changes:

- Fixes attendance page issue. Fixes #177 
- Removes freshman project when it's `Pending` in presentation as it generally means `excused`
- Makes date format on Dashboard no longer show timestamp
- Fixes issue with firefox dropdowns. Fixes #178

### CODE REVIEW

File: conditional/templates/intro_eval_slideshow.html [hunk 1]
```diff
@@ -47,8 +47,9 @@ <h3>{{m['technical_seminars']|length}}</h3>
               </div>
             </div>
           </div>
-        <h4><span class="icon glyphicon glyphicon-{% if m['freshman_project'] == 'Passed' %}ok passed{%else%}remove{% endif %}" aria-hidden="true"></span> Freshman Project</h4>
-        
+
+        <h4>{% if m['freshman_project'] != 'Pending' %}<span class="icon glyphicon glyphicon-{% if m['freshman_project'] == 'Passed' %}ok passed{%else%}remove{% endif %}" aria-hidden="true"></span> Freshman Project{% endif %}</h4>
```
Comment (mbillow) [inline conditional/templates/intro_eval_slideshow.html]: Can you make this three lines to show the layered logic? That would be much easier to read.

**Comment by mbillow** [comment]
Could you look into #178 too?

**Comment by devinmatte** [comment]
@mbillow Fixes #178 as well as layering the logic

**Comment by mbillow** [comment]
@devinmatte Is there a reason this hasn't been merged yet? LGTM

**Pull request merged**